great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3247,7 +3227,7 @@ def _is_usage_stats_enabled() -> bool:
             config.BOOLEAN_STATES = states  # type: ignore[misc] # Cannot assign to class variable via instance
             config.read(config_path)
             try:
-                if not config.getboolean("anonymous_usage_statistics", "enabled"):
```

Comment:
This method needs to be updated so the configparser can look at both `anonymous_usage_statistics` AND `analytics`. We also need to strip out the old env var logic above